Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -577,19 +580,10 @@ open class ChatMessageContentView: _View, ThemeProvider, UITextViewDelegate {
             setNeedsLayout()
         }
 
-        var text = content?.textContent ?? ""
-
-        // Translated text
-        if layoutOptions?.contains(.translation) == true,
-           let currentUserLang = channel?.membership?.language,
-           let translatedText = content?.translatedText(for: currentUserLang) {
-            text = translatedText
-
-            if let languageText = Locale.current.localizedString(forLanguageCode: currentUserLang.languageCode) {
-                translationLabel?.text = L10n.Message.translatedTo(languageText)
-            }
+        var text = content?.text ?? ""
+        if let message = content, let channel = channel {
```

Comment:
Is there a chance the message is there, while the channel is not available?

Reply:
In this case, it is very unlikely. But it is probably a good idea to make it future-proof and make the `ChatChannel` optional like you said above 👍